### ISSUE TITLE
How to use BatchNormalization()?

### ISSUE THREAD

**Issue by Tainyaaaa** [phdong21/CNN4CE#1]
When I try to use SF,there is an error
attributeerror module 'keras.layers.normalization' has no attribute 'Batchnormalization'
I tried to use batch... ,add the import,change the imoirt ,but still has this problem.Can you help me to use this in tran.py,thanks
